usr/sbin/p11sak/p11sak.c

```diff
@@ -2690,6 +2747,43 @@ static void print_remove_cert_help(void)
     printf("\n");
 }
 
+static void print_extract_cert_pubkey_help(void)
+{
+    const struct p11sak_attr *attr;
+
+    printf("ATTRIBUTES (for filtering):\n");
+    for (attr = p11sak_bool_cert_attrs; attr->name != NULL; attr++) {
+        if (attr->settable)
```

Comment:
Actually, for the filtering attributes there should be no check for `attr->settable`. You can filter for all boolean attrs, regardless if they are settable or not.

Other help functions do this right (e.g. list), with the exception of the help functions for the copy functions (both key and cert), where you also can filter by attr and also can set new attrs. So similar to extract-pubkey, just that it are the same attributes (both key attrs or both cert attrs) for filtering and setting.
Not sure how to do this best. Printing the attr list twice sounds strange....

I guess we should print all attrs in the loop (i.e. not only settable), but maybe adjust the help text printed below?